features/search/filters.feature

```diff
@@ -40,13 +40,11 @@ Feature: Filters
     Then I should see "Bilbo Does the Thing"
       And I should not see "A Hobbit's Meandering"
       And I should not see "Roonal Woozlib and the Ferrets of Nimh"
+    When I follow "Clear Filters"
+    Then I should see "3 Works by meatloaf"
     When I press "Fandoms" within "dd.include"
-      And I uncheck "The Hobbit (1)" within "#include_fandom_tags"
-      And I uncheck "Legend of Korra (1)" within "#include_fandom_tags"
-      And I press "Sort and Filter"
-    Then I should see "Roonal Woozlib and the Ferrets of Nimh"
```

Comment:
Let's keep these assertions at the end so we know that "Clear Filters" takes effect:
```gherkin
    Then I should see "Roonal Woozlib and the Ferrets of Nimh"
      And I should see "A Hobbit's Meandering"
      And I should see "Bilbo Does the Thing"
```
The same goes for the exclusion filters scenario as well.